components/org.wso2.micro.integrator.core/src/main/java/org/wso2/micro/integrator/core/TemporaryService.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * you may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.micro.integrator.core;
+
+public class TemporaryService {
```

Comment:
Shall we have a better name here? Just to understand the context. Maybe we can add a Javadoc comment as well.

Reply:
+1 Fixed